backend/commons/src/main/java/org/sonarsource/sonarlint/core/commons/util/git/BlameParser.java

```diff
@@ -48,13 +48,12 @@ public static void parseBlameOutput(String blameOutput, String currentFilePath,
     }
     var currentFileBlame = new BlameResult.FileBlame(currentFilePath, numberOfLines);
     blameResult.getFileBlameByPath().put(currentFilePath, currentFileBlame);
-    var fileSections = blameOutput.split(FILENAME);
+    var lineBlameSnippets = blameOutput.split(FILENAME);
     var currentLineNumber = 0;
 
-    for (var fileSection : fileSections) {
-      if (shouldSkipSection(fileSection, currentLineNumber >= numberOfLines)) {
-        continue;
-      }
+    // because of the way we split the blame output, the last snippet should be skipped
+    for (var i = 0; i < lineBlameSnippets.length - 1; i++) {
+      var fileSection = lineBlameSnippets[i];
```

Comment:
I have the impression it's not only a change about logging but also a change of behavior. There was some kind of filtering before and it'snot the case anymore. Is it expected? If so, could you update the ticket to describe that?

Reply:
Updated the ticket description.